### DISCUSSION TITLE
Welcome to DNN_Tailwind Discussions!

### DISCUSSION THREAD

**Discussion by skrantzman** [skrantzman/DNN_Tailwind#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community that love DNN and TailwindCSS. We hope that's you:
  * Ask questions you’re wondering about.
  * Share your ideas.
  * Make observations
  * Tell us what you like or dislike about DNN_Tailwind
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ajplopez** [comment]
Hey hey! Thanks for putting this together. Exciting to see things moving in this direction and hopefully this encourages other DNN'ers to build their own custom themes. 

As a long time skinner, and new tailwindcss us, I'm excited to see where this goes.
